docs/mp/reactivemessaging/mock.adoc

```diff
@@ -0,0 +1,140 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Mock Connector
+:toc:
+:toc-placement: preamble
+:description: Reactive Messaging Mock connector for testing
+:keywords: helidon, mp, messaging, test, mock
+:feature-name: Mock Connector
+:microprofile-bundle: false
+:rootdir: {docdir}/../..
+
+== ToC
+
+- <<Overview, Overview>>
+- <<Emitting data, Emitting data>>
+- <<Asserting data, Asserting data>>
+- <<Configuration, Configuration>>
+- <<Helidon test with Mock Connector, Helidon Test>>
+
+== Overview
+Mock connector is a simple application scoped bean that can be used for emitting to a channel
+or asserting received data in a test environment. All data received are kept in memory only.
+
+include::{rootdir}/includes/mp.adoc[]
+include::{rootdir}/includes/dependencies.adoc[]
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.messaging.mock</groupId>
+    <artifactId>helidon-messaging-mock</artifactId>
+</dependency>
+----
+
+WARNING: Mock connector should be used in the test environment only!
```

Comment:
Caution: should be used here. We restrict warnings for bodily harm. 

Caution: The mock connector should only be used in test environments.